libs/SalesforceSDKCore/SalesforceSDKCoreTests/ScreenLockManagerTests.m

```diff
@@ -87,15 +87,9 @@ - (void)testLockScreenTriggers {
     XCTAssertTrue([[[SFSDKWindowManager sharedManager] screenLockWindow] isEnabled]);
     
     // Backgrounding/foregrounding on lock -- lock screen should remain & callback shouldn't be called
-    XCTestExpectation *expectation = [[XCTestExpectation alloc] initWithDescription:@"Callback"];
```

Comment:
I think this expectation was added for a bug fix, did the behavior change?

Reply:
Can you please DM me a link to the WI?  I removed this because it causes the test runner to crash.  If you run the tests in xcode it's fine but on the command line (like we do in CI) it has a bunch of side effects that cause other tests to fail.  It took me a long time to figure this out but we weren't seeing it before because of the run order of tests.